apollo/src/main/graphql/com/hedvig/android/owldroid/graphql/embarkStory.graphql

```diff
@@ -137,7 +137,8 @@ query EmbarkStory($name: String!, $locale: String!) {
             }
             skip {
               ...EmbarkLinkFragment
-            }
+            },
```

Comment:
From where are we actually getting the list of insurance providers to show in this thing? 🤔

Reply:
Good questions. That would be from 
```
insuranceProviders(locale: $locale) {
    id
    name
    externalCollectionId
  }
```
I suppose it would be better if this was done in embark and populated on `EmbarkExternalInsuranceProviderAction`.

Reply:
Agreed! I believe we wrote an extension in giraffe for this last time we had to deal with `insuranceProviders`, perhaps we should do the same here as well?